guidelines/MAINTAINERS-guidelines.md

```diff
@@ -6,60 +6,97 @@ grand_parent: Hyperledger TOC
+
+In most repositories, only the "Maintain" GitHub role is used. It is given to all repository Maintainers.
+
+In some repositories, the maintainers may define a list of "scopes" and each Maintainer given one or more of the designated scopes:
+
+- In rare cases, Maintainers in a repository may request that the Hyperledger Foundation GitHub organization administrators enable designating some Maintainers with different GitHub roles, such as "Admin" (more capable than "Maintain") or "Triage" (less capable than "Maintain").
+  - Each elevated GitHub role needed in a repository requires the Hyperledger Foundation GitHub organization administrators to create and maintain an additional GitHub Team.
+  - The Hyperledger Foundation GitHub organization administrators manage a team per Hyperledger Project that includes all contributors to the project. That team is given the "Read" GitHub role in all project repositories. This team need not be documented in the "MAINTAINERS" file.
+- Maintainers **MAY** define Maintainers scope within a repository don't require
+  elevated GitHub privileges. For example, a scope might include hosting
+  community meetings, or contributing to the Hyperledger Project's Quarterly
+  report.
+
+If there is more than the single "Maintainer" scope used in a repository, there **MUST** be a list of the repository specific scopes in the MAINTAINERS file. The list must include the scope name, the definition of the scope, and if applicable, the related GitHub Role and Team for the scope.
+
+| Scope | Definition | GitHub Role | GitHub Team |
+| ----- | ---------- | ----------- | ----------- |
+|       |            |             |             |
+
+## List of Repository Maintainers
+
+Lists of active and emeritus maintainers MUST be included in the `MAINTAINERS` file.
+
+Changes to the maintainers lists MUST be made via Pull Requests. Once a new `MAINTAINERS` file is created or a PR changing the maintainer lists within the file is merged, a corresponding update to the affected GitHub Teams within the Hyperledger GitHub organization must be made. This is a manual process and the maintainers must ensure that it occurs.
+
+It is recommended that the lists be sorted alphabetically and contain at least the maintainers name and GitHub ID. If useful, other columns may be added to the table, such as LFID, Discord ID, Email, and Company Affiliation.
+
+A "Scope" column **MUST** be included if there is more than one scope defined in the repository.
+
+The following shows a suggested format for the information:
 
 **Active Maintainers**
 
-| Maintainer | GitHub ID | LFID | Chat ID | Email | Company Affiliation | Scope |
-| ---------- | --------- | ---- | ------- | ----- | ------------------- | ----- |
-|            |           |      |         |       |                     |       |
```

Comment:
Why are we removing the other fields? The GitHub ID is not always enough to contact the person in question. You are removing the MUST to have chat ID or email. The company affiliation lets us know if we have diversity of maintainers for a project (i.e., that the project is not dependent on a single company to survive).

Reply:
My experience with creating the table is that the information is hard to come by when initially populating the file when adding it to an existing repository.  It’s fine when starting from scratch, but painful when all you to start from is the GitHub Team.  I also think that some of the fields (e.g., LFID) are unnecessary.

AFAIK — the requirement to be able to contact the person is satisfied by the GitHub Id, so the extra methods were redundant.

I had never seen the company affiliation field previously (my bad 😄 ) and thought that would be painful to maintain. Is it being actively used for diversity reporting or is that asperational?

I’m open to adding some of the fields back, but recommend that we have a good reason for adding each one.

Reply:
I think the idea is that the maintainers themselves could populate this information, so I'm not sure it would be difficult to accumulate.  Any active maintainer should have no problems adding themselves to the table.

The LFID is useful for keeping track of people across companies and projects.  More importantly, it's useful for the people themselves, so they can show others what they've done even if they switch jobs or projects.

We are also actively using company affiliation to measure diversity--it is one of the most important contributor diversity metrics.  Lots of people ask us about the breakdowns of contributors by company, and it makes it difficult for us to answer this question if people contribute using their non-company email addresses (which is fine--we, and potential contributors, just want a way to measure company diversity).

Reply:
With the re-addition of these columns, we still have an open item about whether an email address or chat ID should be required. Currently line 56 says that it is recommended.